grails-app/views/layouts/main_component.gsp

```diff
@@ -42,12 +42,11 @@
         }
     }
     </style>
-    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <script src="https://apis.google.com/js/platform.js"></script>
```

Comment:
IIRC, async defer is the recommended approach: https://developers.google.com/+/web/api/javascript